measureme/src/profiling_data.rs

```diff
@@ -306,26 +265,50 @@ impl ProfilingDataBuilder {
     }
 }
 
+impl<'a> ExactSizeIterator for ProfilerEventIterator<'a> {}
+
+fn event_index_to_addr(event_index: usize) -> usize {
```

Comment:
Is this being used?

Reply:
Yes, in `decode_event`. It was used multiple times in a previous version but not anymore it seems. I'll inline it.

Reply:
Nah, it's fine. I just couldn't find where it was used.